src/shared/lib/StoreUtils.ts

```diff
@@ -1627,6 +1627,25 @@ export function getSampleClinicalDataMapByThreshold(
     );
 }
 
+export function getSampleClinicalDataMapByKeywords(
+    clinicalData: ClinicalData[],
+    clinicalAttributeId: string,
+    keywords: string[]
+) {
+    return _.reduce(
+        clinicalData,
+        (acc, next) => {
+            if (next.clinicalAttributeId === clinicalAttributeId) {
+                if (keywords.includes(next.value)) {
+                    acc[next.sampleId] = next;
+                }
+            }
+            return acc;
+        },
+        {} as { [key: string]: ClinicalData }
```

Comment:
i believe if you type the acc argument (1637), instead of here, you can avoid using "as" which is stronger.
```
(acc: {...}, next) => {...}

```